doc/03-Configuration.md

```diff
@@ -101,6 +150,17 @@ There are separate options for the full history tables used to display history i
 SLA tables which store the minimal information required for SLA reporting,
 allowing to keep this information for longer with a smaller storage footprint.
 
+For YAML configuration, the options are part of the `retention` dictionary.
+For environment variables, each option is prefixed with `ICINGADB_RETENTION_`.
+
+When using environment variables, the Retention `options` are formatted similar to the [logging components](#logging-components) from above.
+
+```
+# Environment Variables
+ICINGADB_RETENTION_HISTORY_DAYS=14
+ICINGADB_RETENTION_OPTIONS=comment:356,sla:90
```

Comment:
This will yield `invalid configuration: invalid key sla for history retention`. There's `sla-days` for sla.

Reply:
Wait, shouldn't it be `sla_downtime` or `sla_state`? Then this is also wrong/outdated in the table below.

https://github.com/Icinga/icingadb/blob/0eb1ad8c4f0485a3bc06cb8ed70ed3a9bd672d82/pkg/icingadb/history/retention.go#L110-L114
https://github.com/Icinga/icingadb/blob/0eb1ad8c4f0485a3bc06cb8ed70ed3a9bd672d82/pkg/icingadb/history/retention.go#L79-L94

The `sla-days` should be `ICINGADB_RETENTION_SLA_DAYS`. I am going to alter the change.

Reply:
I have synchronized the retention categories from the `RetentionStatements` to both the docs and the example configuration. Good catch, btw! Please feel free to take another look.

Reply:
Oh wait. I messed up. The two `sla_` keys are of the `RetentionSla` type, not `RetentionHistory`..

Reply:
Now it should be fine. The `sla` entry in `options` seems to be a forgotten artifact and I confused myself after skimming the common code. Please feel free to take a look now :>

Reply:
> Oh wait. I messed up. The two `sla_` keys are of the `RetentionSla` type, not `RetentionHistory`..

Yes, `sla` cannot be further customized. There is only `sla-days`.

> Now it should be fine. The `sla` entry in `options` seems to be a forgotten artifact and I confused myself after skimming the common code. Please feel free to take a look now :>

I checked the example configuration and our code for a leftover, but couldn't find any. Is there anything else to do? Maybe I just misunderstood your comment :).